devices/sonoff/snzb-06p.json

```diff
@@ -0,0 +1,126 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "SONOFF",
+  "modelid": "SNZB-06P",
+  "vendor": "Sonoff",
+  "product": "SNZB-06P",
+  "sleeper": false,
+  "status": "Gold",
+  "path": "/devices/snzb-06p.json",
+  "subdevices": [
+    {
+      "type": "$TYPE_PRESENCE_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0406"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/dark",
+          "refresh.interval": 300,
```

Comment:
As the error suggests, the interval should be greater than the one you've chosen at the end of the DDF for attribute reporting. As you chose 300 seconds below, a value of 360 is recommended here.